packages/ui/src/containers/SetPassword/TogglePassword.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslation } from 'react-i18next';
+
+import Checkbox from '@/components/Checkbox';
+import { onKeyDownHandler } from '@/utils/a11y';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  isChecked?: boolean;
+  onChange: (checked: boolean) => void;
+};
+
+const TogglePassword = ({ isChecked, onChange }: Props) => {
+  const { t } = useTranslation();
+
+  const toggle = () => {
+    onChange(!isChecked);
+  };
+
+  return (
+    <div
+      role="radio"
+      aria-checked={isChecked}
+      tabIndex={0}
+      className={styles.passwordSwitch}
+      onClick={toggle}
+      onKeyDown={onKeyDownHandler({
+        Escape: () => {
+          onChange(false);
+        },
+        Enter: toggle,
+        ' ': toggle,
+      })}
+    >
+      <Checkbox name="toggle-password" checked={isChecked} className={styles.checkBox} />
```

Comment:
nit: `name` prop seems to be useless here

Reply:
for better accessibility and browser hint